setup.py

```diff
@@ -41,7 +41,7 @@ def readme():
 if PKG_NAME == "retworkx":
     README = retworkx_readme_compat + README
     PKG_PACKAGES = ["retworkx"]
-    PKG_INSTALL_REQUIRES.append(f"rustworkx=={PKG_VERSION}")
+    PKG_INSTALL_REQUIRES.append(f"rustworkx>={PKG_VERSION}")
```

Comment:
```suggestion
    PKG_INSTALL_REQUIRES.append(f"rustworkx=={PKG_VERSION}")
```

I am kind of concerned with allowing `retworkx==0.13` and `rustworkx==0.14` in the same system, do you think it is safe?

Reply:
I do, I've been thinking about our long term strategy towards moving away from retworkx permenantly. We've documented in rustworkx 1.0.0 we'll stop carrying around the retworkx shim, but I was thinking there would be no harm in leaving retworkx on pypi with an uncapped requirement it should never break as long as python doesn't change how import works. But I think for this PR you're right we should leave this as is, I'll open up a dedicated PR for this and document my thinking there.